### ISSUE TITLE
MultiMc 5 (windows) keeps getting deleted every time there is a new update.

### ISSUE THREAD

**Issue by TheDorMan** [MultiMC/MultiMC5#276]
Every time, when a new update is avilable- my multimc is deleting itself (i unchecked the auto-update) and everything is getting deleted.
why does it happen and how can i stop this?

**Comment by peterix** [comment]
I'd say 'no'.

**Comment by DahliaSR** [comment]
Yes it is an issue again.
Since today, the stable release for windows, does not work correctly.

I just downloaded mmc-stable-win32.zip.
Extracted with 7zip and executed Multimc.exe.

I got an Error Message, that the file does either notexist or I do not have permission.
I click okay and the Multimc.exe is deleted.

I does not matter to where I extract the zip or if I change permissions on windows, its always the same issue.

The current developrer DOES work correctly for me.
